client/src/pages/private/ParticipantView.js

```diff
@@ -745,7 +818,7 @@ export default () => {
             />
           </Box>
         </Grid>
-        <Menu
+    {(!roles.includes('ministry_of_health')) && <Menu
```

Comment:
Does this prevent superusers from accessing this feature, or are their permissions being handled earlier?

Reply:
This was implemented to avoid a bug that occurred with anchor elements - I'll extend it to superusers.